### ISSUE TITLE
Enchanted Books with multiple enchantments removed

### ISSUE THREAD

**Issue by Bet1K** [Shrek5InTheatres2019/BookShelf#5]
Running into an issue while using bookshelves to store enchanted books. If an enchanted book with more than enchantment (NBT: 2 (or more) tags) is stored, it will remove all enchantments but one and put the NBT tag back to 1.

EDIT: tested with a book with two enchantments and NBT tag 1, still removed an enchantment

**Comment by Shrek5InTheatres2019** [comment]
Yep, this is a known bug, i’ve been working on fixing this for the past bit
